apps/deploy-web/src/components/auth/AuthPage/AuthPage.tsx

```diff
@@ -0,0 +1,292 @@
+                      >
+                        Log in
+                      </span>
+                    </div>
+                  </d.TabsTrigger>
+                  <d.TabsTrigger
+                    value="signup"
+                    className="flex-1 cursor-pointer rounded-none border-0 border-b-2 border-l-0 border-r-0 border-t-0 border-b-transparent bg-transparent py-1.5 shadow-none transition-colors data-[state=active]:border-b-neutral-900 data-[state=active]:bg-transparent data-[state=active]:shadow-none hover:bg-transparent focus-visible:outline-none focus-visible:ring-0 focus-visible:ring-offset-0 dark:data-[state=active]:border-b-neutral-100"
+                  >
+                    <div className="flex items-center justify-center gap-2 px-2.5 py-2">
+                      <span
+                        className={`text-sm font-normal leading-5 transition-colors ${activeTab === "signup" ? "text-neutral-950 dark:text-[var(--text-light)]" : "text-neutral-500 hover:text-neutral-700 dark:text-neutral-400 dark:hover:text-neutral-300"}`}
+                      >
+                        Sign up
+                      </span>
+                    </div>
+                  </d.TabsTrigger>
+                </d.TabsList>
+              </div>
+
+              <d.SocialAuth onSocialLogin={redirectToSocialLogin} />
+
+              <div className="relative flex items-center justify-center self-stretch py-2.5">
+                <Separator className="absolute inset-0 top-1/2" />
+                <div className="current relative top-[-1px] z-10 px-2" style={{ backgroundColor: "hsl(var(--background))" }}>
+                  <span className="relative top-1/2 text-xs font-normal text-neutral-500 dark:text-neutral-400">Or continue with</span>
+                </div>
+              </div>
+
+              {signInOrSignUp.isError && (
+                <d.Alert variant="destructive" className="mb-5">
+                  <d.AlertDescription>
+                    {isHttpError(signInOrSignUp.error) && signInOrSignUp.error.response?.data.message
+                      ? signInOrSignUp.error.response.data.message
+                      : "An unexpected error occurred. Please try again or contact support if the issue persists."}
+                  </d.AlertDescription>
+                </d.Alert>
+              )}
+
+              <d.TabsContent value="login" className="mt-0">
+                <d.SignInForm isLoading={signInOrSignUp.isPending} onSubmit={value => signInOrSignUp.mutate({ type: "signin", value })} />
+              </d.TabsContent>
+
+              <d.TabsContent value="signup" className="mt-0">
+                <d.SignUpForm isLoading={signInOrSignUp.isPending} onSubmit={value => signInOrSignUp.mutate({ type: "signup", value })} />
+              </d.TabsContent>
+            </d.Tabs>
+          </div>
+        </div>
+      </>
+    </d.AuthLayout>
+  );
+}
+
+function TrialIcon() {
```

Comment:
suggestion: these icons should be in separate files in the shared components folder